docs/install.md

```diff
@@ -1,77 +1,197 @@
+    Functions](https://console.cloud.google.com/apis/library/cloudfunctions.googleapis.com)
+*   Follow the
+    [instruction](https://cloud.google.com/pubsub/docs/quickstart-console) to:
+    *   Enable [Cloud
+        Pub/Sub](https://console.cloud.google.com/apis/library/pubsub.googleapis.com)
+    *   Create a new Pub/Sub topic and subscription **(pull type)**
+    *   Please take a note of the topic and subscription name for the
+        configuration steps
+*   Enable [Cloud
+    Datastore](https://console.cloud.google.com/apis/api/datastore.googleapis.com)
+    *   Go to Datastore in the cloud console
+    *   Hit the `Create Entity` button
+    *   Select the region that your data should be in. No need to create any
+        Entities after selecting your region
+*   Enable [Cloud Deployment Manager
+    V2](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
+    for `gcloud`
+
+## Local Turbinia
+
+*   If Turbinia will run on local machines, jump to [Core Installation
+    Steps](#core-installation-steps)
+*   Otherwise, follow [GCP Turbinia](#gcp-turbinia)
+
+## GCP Turbinia
+
+The following is a one possible configuration and setup for Turbinia in GCP.
+This is still a rough process and future versions will be containerized.
+
+*   Create a new GCE server VM from a recent version of Debian or Ubuntu
+    *   This should work on other Linux flavors, but these are untested. Feel
+        free to test and fix them ;)
+*   Follow [Core Installation Steps](#core-installation-steps)
+*   Create a new image from the server VM's disk
+*   Create a new Instance Template using the newly created image
+*   Create a new Managed Instance Group from the newly created Instance Template
+*   Create a new GCS bucket and create new directories for `scripts` and
+    `output/logs`
+*   Mount your GCS bucket on your server VM
+*   cp `turbinia/tools/gcp_init/*.sh` into your locally mounted copy of
+    `$your_bucket/scripts`
+*   Edit the variables in `scripts/start-wrapper.sh` and
+    `scripts/start-turbinia-common.sh` as appropriate
+*   ***NOTE:*** Please note that the `start-wrapper.sh` script has a
+    `GOOGLE_APPLICATION_CREDENTIALS` environment var in the middle of the script
+    that needs to be updated
+*   In your worker VM, add a new custom metadata key `startup-script-url`
+    pointing to `gs://$your_bucket/scripts/start-wrapper.sh`
+*   Upon start, your VM should mount your GCS Bucket, and copy the start scripts
+    into the home directory of the Turbinia user and will then start the
+    Turbinia worker
+*   If you are running in GCP, you may also want to install [GCS
+    FUSE](https://cloud.google.com/storage/docs/gcs-fuse)
+
+## Core Installation Steps
+
+*   Install python-dev, build essential, pip and setuptools
+    *   `sudo apt-get install python-dev build-essential python-setuptools
+        python-pip`
+*   Install virtualenv and git
+    *   `sudo apt-get install python-virtualenv git`
```

Comment:
Can we combine these into the previous apt-get install line?

Reply:
done